tests/unit/store/getters.spec.ts

```diff
@@ -3,97 +3,53 @@ import getters from '@/store/getters';
 import QueryRepresentation from '@/sparql/QueryRepresentation';
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 
+const simpleRootState: RootState = {
+	conditionRows: [ {
+		valueData: { value: 'foo', valueError: null },
+		propertyData: {
+			id: 'P123',
+			label: 'abc',
+			datatype: 'string',
+			isPropertySet: true,
+			propertyError: null,
+		},
+		propertyValueRelationData: { value: PropertyValueRelation.Matching },
+		conditionId: '0.123',
+		subclasses: false,
+		negate: false,
+	} ],
+	limit: 0,
+	useLimit: false,
+	omitLabels: true,
+	errors: [],
+};
+
 describe( 'getters', () => {
 
 	describe( 'limitedSupport', () => {
 		it( 'returns false if the datatype is supported', () => {
-			const state: RootState = {
-				conditionRows: [ {
-					valueData: { value: '', valueError: null },
-					propertyData: {
-						id: 'P123',
-						label: 'abc',
-						datatype: 'string',
-						isPropertySet: true,
-						propertyError: null,
-					},
-					propertyValueRelationData: { value: PropertyValueRelation.Matching },
-					conditionId: '0.123',
-					subclasses: false,
-				} ],
-				omitLabels: true,
-				errors: [],
-				limit: 0,
-				useLimit: false,
-			};
+			const state: RootState = JSON.parse( JSON.stringify( simpleRootState ) );
```

Comment:
That could be done nicer with a method `getFreshRootState(): RootState`, but that can be refactored later. This is already an improvement :pray: